File: 1-modelagem-codigo-php/src/app/Livro.php
<?php

namespace App;

class Livro
{
    private int $id;
    private string $titulo;
    private bool $disponivel;

    public function __construct(int $id, string $titulo, bool $disponivel = true)
    {
        $this->id = $id;
        $this->titulo = $titulo;
        $this->disponivel = $disponivel;
    }

    public function getId(): int
    {
        return $this->id;
    }

    public function getTitulo(): string
    {
        return $this->titulo;
    }

    public function isDisponivel(): bool
    {
        return $this->disponivel;
    }
}

File: 1-modelagem-codigo-python/app/livro.py
class Livro:
    def __init__(self, id: int, titulo: str, disponivel: bool = True):
        self.__id = id
        self.__titulo = titulo
        self.__disponivel = disponivel

    def get_id(self) -> int:
        return self.__id

    def get_titulo(self) -> str:
        return self.__titulo

    def is_disponivel(self) -> bool:
        return self.__disponivel